README.md

```diff
@@ -7,20 +7,43 @@
 [![Latest Unstable Version](https://poser.pugx.org/ttn/tea/v/unstable.svg)](https://packagist.org/packages/ttn/tea)
 [![License](https://poser.pugx.org/ttn/tea/license.svg)](https://packagist.org/packages/ttn/tea)
 
-This TYPO3 extension is an example for writing unit and functional tests for
-Extbase/Fluid-based extensions for TYPO3 CMS using PHPUnit.
+This TYPO3 extension is an example of best practices in continuous integration and automated code checks, also
+writing unit and functional tests for Extbase/Fluid-based extensions for TYPO3 CMS using PHPUnit.
 
 It also is an example for
 [best practices for extbase/fluid](https://github.com/oliverklee/workshop-handouts/tree/main/extbase-best-practices).
 
 For information on the different ways to execute the tests, please have a look
 at the [handout to my workshops on test-driven development (TDD)](https://github.com/oliverklee/tdd-reader).
 
-## Running the unit tests from the command line
+## Feature list
+All of those checks are available in Github Actions and in Gitlab CI.
+####PHP Lint check by php -l
```

Comment:
Some comments concerning the Markdown syntax:

- I propose having a blank line above the heading to make the structure more clear when reading the source.
- You're skipping one heading level here: You're going from `##` to `####` instead of to `###`.